packages/pv-scripts/README.md

```diff
@@ -73,12 +73,30 @@ For further customization of the webpack-config, specific config-files can be ad
 **webpack.config.js:**
 Valid webpack.config files which will be merged with both (dev/prod) default configs.
```

Comment:
shouldnt it be singular: "Valid webpack.config file which will be merged.."?

Reply:
fixed